pkgs/applications/networking/syncthing/default.nix

```diff
@@ -47,7 +47,7 @@ let
         description = "Open Source Continuous File Synchronization";
         changelog = "https://github.com/syncthing/syncthing/releases/tag/v${version}";
         license = licenses.mpl20;
-        maintainers = with maintainers; [ pshendry joko peterhoeg andrew-d ];
+        maintainers = with maintainers; [ joko peterhoeg andrew-d ];
```

Comment:
You dropped yourself as a maintainer here.

Reply:
Ah, I should have called that out as intentional (it is). I've been out of the loop for Syncthing maintenance for some time; I left the others since I'm the only maintainer, so I'll make a best effort on those.

Reply:
Would it be possible to do this in a separate commit, please? :)

Reply:
@cole-h Updated